### ISSUE TITLE
Icon: Proposal to create a new icon for "Post Comments Link"

### ISSUE THREAD

**Issue by t-hamano** [WordPress/gutenberg#40543]
## What problem does this address?

"**Post Comments Link**" block currently uses the same `postCommentsCount`  icon as "**Post Comments Count**" block.

![block](https://user-images.githubusercontent.com/54422211/164710076-f239212c-7b26-4b48-8235-b12793f6d849.png)

I would like to propose that a new icon be created for this block, as we think it is unnatural to have a number displayed on the icon.

## What is your proposed solution?
I can't use design tools, but how about the following icons?
If you have any good ideas, please advise.

![new_icon](https://user-images.githubusercontent.com/54422211/164710262-2f1a2d38-8645-4472-9116-1725d162345a.png)

**Comment by BenjaminZekavica** [comment]
@t-hamano I think the block was deleted, or?

**Comment by t-hamano** [comment]
Hi @BenjaminZekavica,

Thank you for your interest in this issue.

I believe these blocks have not been removed. These blocks are marked as `__experimental: "fse"` in [block.json](https://github.com/WordPress/gutenberg/blob/493d95afd41e37ba8385b6420d8023b11c413dd8/packages/block-library/src/post-comments-link/block.json#L4), so they will not show up in the post editor, only in the site editor.

![inserter](https://github.com/WordPress/gutenberg/assets/54422211/fb26bb79-56fa-4d52-b1da-124700f8c4dc)

**Comment by karmatosed** [comment]
I am not sure about adding extra buttons here considering each is related. Over jumping to add an icon I would like to consider if one to be bother sections works or we want seperate ones. I will note this in PR, this however should have ideally had feedback not design label. We can totally fix that though and let's consider before we go to action.

**Comment by t-hamano** [comment]
@karmatosed Thanks for the feedback.

> I am not sure about adding extra buttons here considering each is related. Over jumping to add an icon I would like to consider if one to be bother sections works or we want seperate ones.

Could you elaborate a bit on this as well? What exactly do you mean by "button"?

**Comment by karmatosed** [comment]
Sorry I meant to type icon and typed button.

**Comment by t-hamano** [comment]
> Over jumping to add an icon I would like to consider if one to be bother sections works or we want seperate ones.

Sorry, I don't know what this actually means 🤔 Can you tell me what specific approach you are referring to?

**Comment by karmatosed** [comment]
What I meant to say was this @t-hamano 

> I am not sure about adding extra icons here considering each is related. Over jumping to add an icon I would like to consider if one to be bother sections works or we want seperate ones.

This refers to even changing the icons. I don’t know if splitting them out makes sense.

**Comment by t-hamano** [comment]
I see. So, you mean that one approach would be to use the same icons for comment-related blocks as well, right? For example:

<img width="368" height="326" alt="Image" src="https://github.com/user-attachments/assets/2cfe8a75-6729-446f-8450-eb0bab91a4ce" />

**Comment by karmatosed** [comment]
Yes, I’m curious about having so many for a similar concept. I’m going to loop in @jasmussen here to get some insights as I really want to get some consistency for iconography.

**Comment by jasmussen** [comment]
I can't recall this Monday morning what other icons share visual DNA as being part of a group like this, but I could swear there was such a group of icons recently where we decided to make more or less an icon template that included a small version of the main icon itself, and then a secondary element. I feel like it wasn't these two:

<img width="101" height="47" alt="Image" src="https://github.com/user-attachments/assets/7100cb56-2614-470d-94aa-34688b8d6c18" />

Nevertheless the example sort of works, though it's perhaps hard to add a link silhouette inside that tiny comment shape. An idea _could_ be to leverage the "Comments" template that has two bubbles, but instead of the second bubble, replace it with a count, a hyperlink icon, etc. 

I don't have bandwidth this week to work on this, alas!
